src/Lykke.Common.Validation/ImageTypes/Fluent/RecoverySelfieImageIFormFileValidator.cs

```diff
@@ -0,0 +1,25 @@
+﻿using Microsoft.AspNetCore.Http;
+
+namespace Lykke.Common.Validation.ImageTypes.Fluent
+{
+    /// <summary>
+    ///     Deep recovery selfie image validator, based on <see cref="ImageTypeIFormFileValidator" />.
+    ///     Use only on <see cref="IFormFile" />.
+    ///     Extensions could not be passed to this validator as parameters.
+    ///     Valid file extensions: ".jpg", ".jpeg", ".png", ".gif", ".bmp".
+    ///     Any other extensions are invalid.
+    /// </summary>
+    public class RecoverySelfieImageIFormFileValidator : ImageTypeIFormFileValidator
```

Comment:
As for me it is not a library method. It is project specific validation.

It is better to provide way to reuse basics from this library and extend in concrete project.

Reply:
Also haven't found test for this method.

Reply:
I assume test for RecoverySelfieImageIFormFileValidator is not needed as it is based on ImageTypeIFormFileValidator for which tests are provided.
If wee need test for every class then I'll add them.